modules/affile/affile-dest.c

```diff
@@ -97,7 +97,7 @@ struct _AFFileDestWriter
   time_t last_msg_stamp;
   time_t last_open_stamp;
   time_t time_reopen;
-  struct iv_timer reap_timer;
+  struct iv_timer retry_timer;
```

Comment:
I think I was misunderstood here and I may have misunderstood the patch
while reading it. So I agree with @juhaszviktor that it should use a
separate timee.

-- 
Bazsi

On Thu, May 28, 2015 at 4:22 PM, Nagy Gergő <notifications@github.com>
wrote:

> In modules/affile/affile-dest.c
> <https://github.com/balabit/syslog-ng/pull/444#discussion_r31237792>:
>
> > @@ -97,7 +97,7 @@ struct _AFFileDestWriter
> >    time_t last_msg_stamp;
> >    time_t last_open_stamp;
> >    time_t time_reopen;
> > -  struct iv_timer reap_timer;
> > +  struct iv_timer retry_timer;
>
> My previous solution was using two timers but @bazsi
> <https://github.com/bazsi> is right and adding a redundant timer when the
> other one is not active is a waste per file destination. You're right about
> the multiple responsibility. Although, I think the responsibility of a
> timer is to be shot at the given time. The handlers split the functionality
> not the timers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/444/files#r31237792>.
>